sklearn/linear_model/tests/test_omp.py

```diff
@@ -18,8 +18,11 @@
 from sklearn.datasets import make_sparse_coded_signal
 
 n_samples, n_features, n_nonzero_coefs, n_targets = 25, 35, 5, 3
-y, X, gamma = make_sparse_coded_signal(n_targets, n_features, n_samples,
-                                       n_nonzero_coefs, random_state=0)
+y, X, gamma = make_sparse_coded_signal(n_samples=n_targets,
+                                       n_components=n_features,
+                                       n_features=n_samples,
```

Comment:
This one is a bit weird, was the test correct in the first place? Should we change the variables names?

Reply:
I'm not sure if it was correct, my gut feeling is that no. But this still passes the test.

Reply:
If it passes when re-ordering the args then we should rather do that

I can take over if you're busy with other release stuff, LMK

Reply:
I'd be happy if you take over :)

Reply:
Well, I tried. A few asserts fail in the test even after switching the values. Just gonna approve as is then